README.md

```diff
@@ -21,6 +21,16 @@ visibility into status and actions taken to address a complaint.
 * Run the server: `bundle exec rails s`
 * Visit the site: http://localhost:8080
 
+#### Local Configuration
+
+Environment variables can be set in development using the [dotenv](https://github.com/bkeepers/dotenv) gem.
+
+Consistent but sensitive credentials should be added to `config/credentials.yml.env` by using `$ rails credentials:edit`
+
+Any changes to variables in `.env` that should not be checked into git should be set
+in `.env.local` and/or `.env.test.local`.
+Both files are required because `.env.local` is loaded for all environments _except_ test.
```

Comment:
[question] should we be setting local envs for tests from a file, or should we be setting them when they are different within the tests themselves?

Reply:
Well... certainly within the tests for anything related to the test or that should persist. I was thinking more along the lines of "how do I override something short term to debug" but maybe that's not even worth calling out for `RAILS_ENV=test`.